pegasus/sites.v3/hourofcode.com/views/home_signup.haml

```diff
@@ -4,7 +4,7 @@
 %a#hocsignupform{name: 'signup'}
 #join
   %h1#join-us-header=hoc_s(:front_join_us_button)
-  -if ["post-hoc", false].include?(hoc_mode)
+  -if true #["post-hoc", false].include?(hoc_mode)
```

Comment:
for my own edification: why was this necessary, if `hoc_mode` is currently supposed to be `actual-hoc`?